tests/test-4.0.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Do we need the file is test is commented?

Reply:
I 'll remove it.